samples/exposed-spring/README.md

```diff
@@ -0,0 +1,11 @@
+# Exposed-Spring
+
+This Spring Boot 3 based project uses Exposed for CRUD (Create, Read, Update, Delete) operations.
```

Comment:
Could you also add a short description about the main classes? I think `UserController` and `UserService` should be enough, +/- `SchemaInitialize` and `SpringApplication` maybe. Here's how it was formatted for [the ktor sample](https://github.com/JetBrains/Exposed/blob/main/samples/exposed-ktor/README.md).